engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java

```diff
@@ -21,7 +20,6 @@ final class EpbContext {
   private final boolean isInner;
   private final TruffleLanguage.Env env;
   private @CompilationFinal TruffleContext innerContext;
-  private final ReentrantLock lock = new ReentrantLock();
```

Comment:
Less locks is better, indeed.

Reply:
Yes, especially when this one was not used anywhere. I have just noticed it when I was browsing the code base.